### PR TITLE
Add new keyboad(macropad) Acperience12

### PR DESCRIPTION
**Pull request by yynmt**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by yynmt** [comment]
I created a readme.md under the rev1 directory, why doesn't the check succeed?

```
linting acperience12/rev1
☒ Missing keyboards/acperience12/rev1/readme.md
☒ Lint check failed!
```

File: keyboards/yynmt/acperience12/acperience12.c [hunk 1]
```diff
@@ -0,0 +1 @@
+#include "acperience12.h"
```
Comment (drashna) [inline keyboards/yynmt/acperience12/acperience12.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/yynmt/acperience12/acperience12.h [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#pragma once
```
Comment (drashna) [inline keyboards/yynmt/acperience12/acperience12.h:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/yynmt/acperience12/acperience12.h [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+
+#ifdef KEYBOARD_yynmt_acperience12_rev1
+  #include "rev1.h"
```
Comment (drashna) [inline keyboards/yynmt/acperience12/acperience12.h:4]:
```suggestion
#    include "rev1.h"
```

File: keyboards/yynmt/acperience12/keymaps/default/keymap.c [hunk 4]
```diff
@@ -0,0 +1,25 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/yynmt/acperience12/keymaps/default/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/yynmt/acperience12/keymaps/default/keymap.c [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+    KC_5, KC_6,
+    KC_7, KC_8,
+    KC_9, KC_A,
+    KC_B, KC_C
+  ),
+};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  return true;
+}
```
Comment (drashna) [inline keyboards/yynmt/acperience12/keymaps/default/keymap.c:25]:
```suggestion
```

File: keyboards/yynmt/acperience12/rev1/config.h [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+
+#pragma once
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x04D8 //Sublicense from Microchip Technology
+#define PRODUCT_ID      0xEA51 //Sublicense from Microchip Technology
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    yynmt
+#define PRODUCT         Acperience12
+#define DESCRIPTION     12keys macropad
```
Comment (drashna) [inline keyboards/yynmt/acperience12/rev1/config.h:25]: This is no longer used. 
```suggestion
```

File: keyboards/yynmt/acperience12/rev1/rev1.h [hunk 7]
```diff
@@ -0,0 +1,22 @@
+#pragma once
```
Comment (drashna) [inline keyboards/yynmt/acperience12/rev1/rev1.h:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/yynmt/acperience12/rev1/rev1.c [hunk 8]
```diff
@@ -0,0 +1 @@
+#include "acperience12.h"
```
Comment (drashna) [inline keyboards/yynmt/acperience12/rev1/rev1.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

```suggestion
#include "rev1.h"
```

File: keyboards/yynmt/acperience12/rev1/rev1.h [hunk 9]
```diff
@@ -0,0 +1,22 @@
...
+
+#include "acperience12.h"
+#include "quantum.h"
```
Comment (drashna) [inline keyboards/yynmt/acperience12/rev1/rev1.h:4]:
```suggestion
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Comment by yynmt** [comment]
@drashna Thank you for your review.

File: keyboards/yynmt/acperience12/readme.md [hunk 10]
```diff
@@ -0,0 +1,19 @@
...
+
+![Acperience12](https://raw.githubusercontent.com/yynmt/Acperience12/main/images/main_image.jpg)
+
+Acperience12 is 12 keys macropad.
+
+* Keyboard Maintainer: [yynmt](https://github.com/yynmt)
+* Hardware Supported: The A12 PCB
+* Hardware Availability: [This repository](https://github.com/yynmt/Acperience12) has the plate files and other files.
+
+Make example for this macropad (after setting up your build environment):
```
Comment (drashna) [inline keyboards/yynmt/acperience12/readme.md:11]: Could you add instructions on how to enter the bootloader?

Comment (yynmt) [reply keyboards/yynmt/acperience12/readme.md:11]: How do I describe it?
It doesn't seem to be described in the readme.md of other keyboards.
To enter the bootloader, just press the reset button.
This is the same as the other keyboards.

Comment (drashna) [reply keyboards/yynmt/acperience12/readme.md:11]: It's a newer requirement that we've added.  Basically, where the reset button is located, or if yu have bootmagic enabled, what key is it.  That sort of information.

File: keyboards/yynmt/acperience12/info.json [hunk 11]
```diff
@@ -0,0 +1,20 @@
+{
+    "keyboard_name": "Acperience12",
+    "url": "http://yynmt.com",
+    "maintainer": "yynmt",
+    "width": 2,
+    "height": 6,
+    "layouts": {
+        "LAYOUT": {
+            "key_count": 12,
```
Comment (zvecr) [inline keyboards/yynmt/acperience12/info.json:9]:
```suggestion
```

File: keyboards/yynmt/acperience12/keymaps/default/keymap.c [hunk 12]
```diff
@@ -0,0 +1,37 @@
...
+ */
+
+#include QMK_KEYBOARD_H
+
+// Defines names for use in layer keycodes and the keymap
+enum layer_names {
+  _BASE = 0,
+  _LOWER,
+  _RAISE,
+  _ADJUST
```
Comment (zvecr) [inline keyboards/yynmt/acperience12/keymaps/default/keymap.c:24]:
```suggestion
  _BASE
```

File: keyboards/yynmt/acperience12/rules.mk [hunk 13]
```diff
@@ -0,0 +1,25 @@
...
+NKRO_ENABLE = no            # USB Nkey Rollover
+MIDI_ENABLE = no            # MIDI controls
+AUDIO_ENABLE = no           # Audio output on port C6
+UNICODE_ENABLE = no         # Unicode
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
+HD44780_ENABLE = no 		    # Enable support for HD44780 based LCDs
+
+DEBUG_ENABLE = no
+CUSTOM_MATRIX = no         # Use custom matrix code
```
Comment (zvecr) [inline keyboards/yynmt/acperience12/rules.mk:23]:
```suggestion
# Build Options
#   change yes to no to disable
#
BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
MOUSEKEY_ENABLE = yes       # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = no            # USB Nkey Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
BLUETOOTH_ENABLE = no       # Enable Bluetooth
AUDIO_ENABLE = no           # Audio output
```

**Review by zvecr — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

**Comment by yynmt** [comment]
@zvecr Perhaps I have corrected what you reviewed.
Could you please check it?

**Pull request closed without merging**